dashboard/lib/services/curriculum_pdfs.rb

```diff
@@ -111,14 +112,9 @@ def self.generate_pdfs?(script_data)
       !timestamps_equal(new_timestamp, existing_timestamp)
     end
 
-    # We do not want to generate an overview pdf in a couple cases:
-    # 1) There are no lesson plans in the unit
-    # 2) The unit's published state is in-development or pilot. This is because
-    # we rely on being able to see the unit overview page as a signed out user
-    # in order to generate the overview pdf. When a course is in-development or pilot
-    # signed out users can not see the unit overview page
```

Comment:
I would suggest moving bullet (2) over to the comment for `generate_pdfs?`, since that comment already tries to list out all of the conditions that method is evaluating